Library/Homebrew/livecheck/livecheck.rb

```diff
@@ -612,6 +612,9 @@ def self.latest_version(
       referenced_package = referenced_formula_or_cask || formula_or_cask
 
       livecheck_url_string = livecheck_url_to_string(livecheck_url, referenced_package) if livecheck_url
+      if livecheck_url.is_a?(Symbol) && !livecheck_url_string
+        raise ArgumentError, "`url #{livecheck_url.inspect}` does not reference a checkable URL"
```

Comment:
This feels like it would be nicer to:
- move this logic to an `else` in `livecheck_url_to_string`
- make `livecheck_url_to_string` always return a `String` rather than a `T.nilable(String)`

As a general rule/concept: whenever you can remove `T.nilable` usage and `raise` instead: it's nicer to do so when using Sorbet.

Reply:
Makes sense to me and I agree that's a nicer approach. Thanks for the feedback.